documentation/src/test/java/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java

```diff
@@ -874,6 +874,27 @@ void simpleQueryString() {
 		} );
 	}
 
+	@Test
+	void queryString() {
+		withinSearchSession( searchSession -> {
+			System.err.println(
+					searchSession.search( Book.class )
+							.where( f -> f.queryString().field( "title" ).matching( "\"robots of dawn\"" ) ).toQuery()
+							.queryString()
+			);
+			System.err.println( searchSession.search( Book.class )
+					.where( f -> f.queryString()
+							.field( "title" ).boost( 20.0f )
+							.field( "description" ).boost( 2.0f )
+							.matching( "robots" )
+							.minimumShouldMatch()
+							.ifMoreThan( 0 )
+							.thenRequireNumber( 2 )
+							.end()
+					).toQuery().queryString() );
+		} );
```

Comment:
Adding a comment so that you don't forget to adapt/remove this.

Also, you'll probably want to use a more complicated example for the documentation. Since the main advantage of `queryString` over `simpleQueryString` is that it's more powerful.